tests/test_nanopub.py

```diff
@@ -30,6 +30,29 @@ def test_construction_with_bnode_introduced_concept(self):
         )
         assert str(publication.introduces_concept) == test_concept_uri
 
+    def test_construction_with_derived_from_as_list(self):
+        """
+        Test Publication construction from assertion where derived_from is a list.
+        """
+        test_uri = 'http://www.example.com/my-nanopub'
+        derived_from_list = [   'http://www.example.com/another-nanopub', # This nanopub is derived from several sources
+                                'http://www.example.com/and-another-nanopub',
+                                'http://www.example.com/and-one-more' ]
+        assertion_rdf = rdflib.Graph()
+        assertion_rdf.add((rdflib.term.BNode('DrBob'),
```

Comment:
There's a constant for this assertion on line 8 :)

Reply:
Ah, I got this from ```test_construction_with_bnode_introduced_concept()```, should that one also use ```TEST_ASSERTION```?

Reply:
Actually ```test_from_assertion()``` is also not using ```TEST_ASSERTION```. I've updated all these test functions to use it.